Modules/Devices/src/Devices.Application/Identities/Commands/CancelDeletionProcessAsSupport/CancelDeletionAsSupportCommand.cs

```diff
@@ -4,10 +4,10 @@ namespace Backbone.Modules.Devices.Application.Identities.Commands.CancelDeletio
 
 public class CancelDeletionAsSupportCommand : IRequest<CancelDeletionAsSupportResponse>
 {
-    public CancelDeletionAsSupportCommand(string identityAddress)
+    public CancelDeletionAsSupportCommand(string deletionProcessId)
     {
-        IdentityAddress = identityAddress;
+        DeletionProcessId = deletionProcessId;
     }
 
-    public string IdentityAddress { get; set; }
+    public string DeletionProcessId { get; set; }
```

Comment:
Should additionally require the address to be passed.

Reply:
Changes made.